crates/typos-dict/assets/top-1000-most-frequent-words.csv

```diff
@@ -0,0 +1,1000 @@
+the
```

Comment:
Let's leave this off for now because we'd need to workout cases like "extrememe"

Reply:
I worked these cases out by adding the check:

```rs
                        if only_correction.ends_with(suffix) {
                            // We still want to correct e.g. "extrememe" to "extreme".
                            return true;
                        }
```

Reply:
I think I'd still prefer not be constrained by this very mechanical process.  It can provide insight but I don't trust it to automatically be applied

Reply:
The current process now seems to make exactly the changes we want to all our 63,200 entries, which does inspire some confidence in me. Besides the process is very easy to adapt so I think we can just do so when we figure out that it too eagerly filters out something.

Reply:
> The current process now seems to make exactly the changes we want to all our 63,200 entries

Except this isn't exactly what I want (see the other thread).  Arbitrarily combining words that don't make sense when combined leads to us losing corrections we would have otherwise.